### DISCUSSION TITLE
WebSockets ping frames

### DISCUSSION THREAD

**Discussion by marcuslindblom** [dotnet/aspnetcore#25186] [General]
What is the status on websockets and the hole keep alive thing? Does it work and is it possible to see the ping frames using Chrome, Wireshark or some other tool?

**Comment by davidfowl** [comment]
What's the question?

**Reply by marcuslindblom** [reply]
@davidfowl i read about problems [here](https://mcguirev10.com/2019/08/18/minimal-full-feature-kestrel-websocket-server.html#websocket-ping-pong) and just wanted to know if this works now and is something I can rely on in my application? Also, how can I verify that it work because the frames does not seem to be visible in Chrome?

**Reply by davidfowl** [reply]
I read that article and it hasn't given me any clarity on what the question is. What are you trying to achieve? What is blocking you? What environment are you running in?
